include/RMGHardwareMessenger.hh

```diff
@@ -18,22 +18,36 @@
 
 #include "G4UImessenger.hh"
 
+/** @brief Class for handling custom messenger commands. */
 class RMGHardware;
 class RMGHardwareMessenger : public G4UImessenger {
 
   public:
 
+    /** @brief Constructor based on a @c RMGHardware pointer. */
```

Comment:
I do not see the value in just having doc comments for the sake of having them... This commnet is just 100% the function signature.